cmd/image/qcow2ova/prep/loop.go

```diff
@@ -104,6 +105,31 @@ func mount(opts, src, target string) error {
 	return nil
 }
 
+// checkFileExists return True if file with the name exists else False
+func checkFileExists(name string) (bool, error) {
+	matches, err := filepath.Glob(name)
+	return len(matches) > 0, err
+}
+
+// bootDeviceuuid takes fstab file path as input and returns uuid of boot device
+func bootDeviceuuid(file string) (string, error) {
+	getBootPartitionUUIDcmd := fmt.Sprintf("cat %s | grep /boot | awk '{print $1}'", file)
+	exitcode, out, err := utils.RunCMD("bash", "-c", getBootPartitionUUIDcmd)
```

Comment:
instead of cat and pipe, don't we have an option of opening that file and parse?

Reply:
Done.